apps/publikator/components/editor/MdastPage.js

```diff
@@ -700,12 +701,8 @@ export class EditorPage extends Component {
       isTemplate: isNew ? isTemplate === 'true' : data?.repo?.isTemplate,
       message: message,
       document: {
-        content: parse(
-          stringify(
-            JSON.parse(
-              JSON.stringify(this.editor.serializer.serialize(editorState)),
-            ),
-          ),
+        content: JSON.parse(
+          JSON.stringify(this.editor.serializer.serialize(editorState)),
```

Comment:
this is just to make a deep copy. we are stringify-parsing to JSON, not markdown. but i understand that it offends you and we should find a more transparent solution.

Reply:
But do we even need to make a copy here? It's just used as an argument for the mutation …

Reply:
The only reason that I can think of why this could be necessary is if serialize() doesn't return a plain JSON object?

Reply:
It is dangerous to ask oneself this kind of questions, Jeremy. (Aka: not sure why it's here, perhaps it's just a remnant from the markdown stringify-parsing – I'll investigate.)

Reply:
i think it was because stringifying to markdown was not playing nice with slate, so we needed to execute that operation on a copy of the editor state... so you are probably right, we don't need it. Will try.